layers/generate_layers.py

```diff
@@ -593,11 +600,10 @@ def cleanUpGeojson(path, propertiesToKeep, addIds=False):
 
         counter = 1
         for feature in gj["features"]:
-            # Only keep one property
             keptProperties = {}
             for property in propertiesToKeep:
-                valueToKeep =  feature["properties"].get(property) 
-                if(valueToKeep):
+                valueToKeep = feature["properties"].get(property)
+                if (valueToKeep):
```

Comment:
The () are are redundant by the way, I think black / some linters might remove that kind of thing automatically